pkg/controllers/user/alert/watcher/node.go

```diff
@@ -138,12 +138,11 @@ func (w *NodeWatcher) checkNodeCondition(alert *v3.ClusterAlert, machine *v3.Nod
 
 func (w *NodeWatcher) checkNodeMemUsage(alert *v3.ClusterAlert, machine *v3.Node) {
 	alertID := alert.Namespace + "-" + alert.Name
-	if v3.NodeConditionReady.IsTrue(machine) {
+	if v3.NodeConditionProvisioned.IsTrue(machine) {
```

Comment:
@aiwantaozi could you describe the need for this change? Why do we need to check node mem usage when the cluster might not be ready? `NodeConditionReady` is false when the cluster can't be reached

Reply:
The node(machine) crd will not change to the condition ready, the condition ready is for the k8s original node